setup.py

```diff
@@ -9,7 +9,7 @@
 from distutils.sysconfig import get_config_vars
 
 if sys.platform == 'win32':
-    libraries = ["arb", "flint", "mpir", "mpfr", "pthreads"]
+    libraries = ["arb", "flint", "mpfr", "gmp"]
```

Comment:
I think I'll peg this to an environment variable for now. Longer term I think we want a build system that doesn't use setup.py or distutils at all and that probably means that exactly how it would be configured would be quite different.